### PR TITLE
Replace ns/sriov with ns/ sriov-network-operator

### PR DESCRIPTION
**Pull request by mmirecki**

### CODE REVIEW

**Comment by pkliczewski** [comment]
@mmirecki please make DCO happy

**Comment by masayag** [comment]
@mmirecki please rebase and sign-off the commit (git commit -s)

File: collection-scripts/gather [hunk 1]
```diff
@@ -7,7 +7,7 @@ resources=()
 resources+=(ns/openshift-cnv ns/kubevirt-hyperconverged ns/openshift-operator-lifecycle-manager ns/openshift-marketplace)
 
 # KubeVirt network related namespaces
-resources+=(ns/kubemacpool-system ns/linux-bridge ns/openshift-sdn ns/cluster-network-addons-operator ns/nmstate ns/sriov)
+resources+=(ns/kubemacpool-system ns/linux-bridge ns/openshift-sdn ns/cluster-network-addons-operator ns/nmstate sriov-network-operator)
```
Comment (phoracek) [inline collection-scripts/gather]: should not it be ns/sriov-network-operator?

**Comment by pkliczewski** [comment]
@mmirecki do you still want to work on this PR?

File: collection-scripts/gather (hunk 1 above)
Comment (mmirecki) [reply collection-scripts/gather]: Yes
Done

**Comment by masayag** [comment]
@phoracek please review

**Pull request merged**